lib/plugins/aws/deploy/compile/functions/tests/index.js

```diff
@@ -358,35 +358,40 @@ describe('AwsCompileFunctions', () => {
       ).to.deep.equal(compiledFunction);
     });
 
-    it('should create corresponding function output objects', () => {
+    it('should include description if specified', () => {
```

Comment:
This test does more than just testing for the description, as it includes memory, runtime and compares and the name. Should only compare that `awsCompileFunctions.serverless.service.provider.compiledCloudFormationTemplate.Resources.FuncLambdaFunction.Properties.Description` matches the description.